Source/WebCore/platform/network/curl/CurlMultipartHandle.cpp

```diff
@@ -79,12 +79,12 @@ CurlMultipartHandle::CurlMultipartHandle(CurlMultipartHandleClient& client, CStr
 {
 }
 
-void CurlMultipartHandle::didReceiveMessage(const SharedBuffer& buffer)
```

Comment:
Recently, WebKit is using `std::span<uint8_t>` for this purpose.

Reply:
Fixed